common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -770,10 +804,139 @@ def test_discussion_on_other_team_page(self, is_staff):
+                    num_members=num_members,
+                    max_size=max_size,
+                    members_text='Member' if num_members == max_size else 'Members'
+                ),
+                'team_location': 'Afghanistan',
+                'team_language': 'Afar'
+            }
+        )
+        self.assertEqual(self.team_page.team_members, num_members)
+
+        if is_member:
+            self.assertEqual(self.team_page.team_membership_text, 'You are a member of this team.')
+            self.assertTrue(self.team_page.team_leave_link_present)
+            self.assertTrue(self.team_page.team_invite_section_present)
+            self.assertEqual(self.team_page.team_invite_help_text, invite_text)
+        else:
+            self.assertEqual(self.team_page.team_membership_text, '')
+            self.assertFalse(self.team_page.team_leave_link_present)
+            self.assertFalse(self.team_page.team_invite_section_present)
+
+    def test_team_member_can_see_full_team_details(self):
+        """
+        Scenario: Team member can see full info for team.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am a member
+        When I visit the Team page for that team
+        Then I should see a full team detail
+        And I should see the team members
+        And I should see the team membership text
+        And I should see the language & country
+        And I should see the Leave Team and Invite Team
+        """
+        self._team_config()
+        self.team_page.visit()
+
+        self.assert_team_details(
+            num_members=1,
+            invite_text='Send this link to friends so that they can join too.'
+        )
+
+    def test_other_users_can_see_limited_team_details(self):
+        """
+        Scenario: Users who are not member of this team can only see limited info for this team.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am not a member
+        When I visit the Team page for that team
+        Then I should not see full team detail
+        And I should see the team members
+        And I should not see the team membership text
+        And I should not see the Leave Team and Invite Team
+        """
+        self._team_config(create_membership=False)
+        self.team_page.visit()
+
+        self.assert_team_details(is_member=False, num_members=0)
+
+    def test_user_can_navigate_to_members_profile_page(self):
+        """
+        Scenario: User can navigate to profile page via team member profile image.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am a member
+        When I visit the Team page for that team
+        Then I should see a profile image for team member
+        When I click on profile image
```

Comment:
"Then I should see profile images for the team members"
"When I click on the first profile image"